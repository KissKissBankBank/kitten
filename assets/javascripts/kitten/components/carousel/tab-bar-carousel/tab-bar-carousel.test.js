import React from 'react'
import renderer from 'react-test-renderer'
import { NavTabCarousel } from './index'

describe('<NavTabCarousel />', () => {
  let component

  beforeEach(() => {
    component = renderer
      .create(
        <NavTabCarousel>
          <NavTabCarousel.Prev>Foobar 1</NavTabCarousel.Prev>
          <NavTabCarousel.Pagination activeIndex={2} />
          <NavTabCarousel.Next>Foobar 2</NavTabCarousel.Next>
        </NavTabCarousel>,
      )
      .toJSON()
  })

  it('matches with snapshot', () => {
    expect(component).toMatchSnapshot()
  })
})
