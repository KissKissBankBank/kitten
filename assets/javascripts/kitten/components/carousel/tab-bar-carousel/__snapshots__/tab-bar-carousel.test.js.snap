// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<NavTabCarousel /> matches with snapshot 1`] = `
<div
  className="tab-bar-carousel__NavTabCarouselStyles-sc-1vfblk9-0 cuQfFM"
>
  <div
    className="prev__LeftArrowContainerStyles-n0wx91-0 kRykYK"
  >
    <svg
      className=""
      data-radium={true}
      fill="#fff"
      height="5.64"
      style={
        Object {
          "transform": "rotate(-90deg)",
        }
      }
      viewBox="0 0 8.48 5.64"
      width="8.48"
      xmlns="http://www.w3.org/2000/svg"
    >
      <path
        d="M0 4.24 L4.24,0 L8.48,4.24 L7.08,5.64 L4.24,2.77 L1.4,5.6 z"
      />
    </svg>
    <div
      className="prev__TextStyles-n0wx91-1 jjcWqd"
    >
      Foobar 1
    </div>
  </div>
  <div
    className="pagination__PaginationStyles-sc-1u3i1hn-1 gEkWnL"
  >
    <div
      className="pagination__BulletPointStyles-sc-1u3i1hn-0 lnKoTT"
    />
    <div
      className="pagination__BulletPointStyles-sc-1u3i1hn-0 dBffWO"
    />
    <div
      className="pagination__BulletPointStyles-sc-1u3i1hn-0 lnKoTT"
    />
  </div>
  <div
    className="next__RightArrowContainerStyles-bcawu0-0 dQpkth"
  >
    <div
      className="next__TextStyles-bcawu0-1 hNVLRJ"
    >
      Foobar 2
    </div>
    <svg
      className=""
      data-radium={true}
      fill="#fff"
      height="5.64"
      style={
        Object {
          "transform": "rotate(90deg)",
        }
      }
      viewBox="0 0 8.48 5.64"
      width="8.48"
      xmlns="http://www.w3.org/2000/svg"
    >
      <path
        d="M0 4.24 L4.24,0 L8.48,4.24 L7.08,5.64 L4.24,2.77 L1.4,5.6 z"
      />
    </svg>
  </div>
</div>
`;
